### ISSUE TITLE
Outdated server lists with Opportunistic IPsec

### ISSUE THREAD

**Issue by comburent** [libreswan/libreswan#1144]
Following the instructions at <https://libreswan.org/wiki/HOWTO:_Opportunistic_IPsec_using_LetsEncrypt>, the hosts listed there seem to no longer have Opportunistic IPsec enabled. (unless this is a problem from my end?)

**Comment by paulwouters** [comment]
yes the servers are currently not properly running - they were kind of forgotten. We should re-enable them again
